pimd/pim_ssm.h

```diff
@@ -19,7 +19,11 @@
 #ifndef PIM_SSM_H
 #define PIM_SSM_H
 
+#if PIM_IPV == 4
 #define PIM_SSM_STANDARD_RANGE "232.0.0.0/8"
+#else
+#define PIM_SSM_STANDARD_RANGE "FF3x::/96"
```

Comment:
This is not a valid prefix...

Reply:
PIM_SSM_STANDARD_RANGE is used here only for displaying purpose in show cli